src/EPR.Calculator.Frontend/Controllers/UploadCSVErrorController.cs

```diff
@@ -10,12 +9,14 @@ public IActionResult Index()
         {
             var listErrorViewModel = new List<ErrorViewModel>
             {
-                new() { DOMElementId = string.Empty, ErrorMessage = "Invalid Entry" },
-                new() { DOMElementId = string.Empty, ErrorMessage = "Invalid Entry1" },
+                new() { DOMElementId = string.Empty, ErrorMessage = "Enter the scheme administrator operating costs for England" },
+                new() { DOMElementId = string.Empty, ErrorMessage = "Enter the communication costs for steel" },
```

Comment:
There should not be any data validation on Front end

Reply:
@udaydefra - this is a mock object to display errors in the UI. Once the integration is done, these will be the errors returned by the API, and we will delete this code.